### ISSUE TITLE
Add python to Readme build instructions

### ISSUE THREAD

**Issue by hodgman** [KhronosGroup/SPIRV-Tools#2407]
I'm following the build instructions and immediately hit with python runtime errors that I'm now diving into. The build instructions don't even mention that having python installed is a prerequisite / dependency of the build system -- can the build documentation be updated to list the requirements of the build environment?

```
8>------ Build started: Project: core_tables, Configuration: Release x64 ------
8>Building Custom Rule D:/projects/SPIRV-Tools-src/source/CMakeLists.txt
8>CMake does not need to re-run because D:\projects\SPIRV-Tools-src\build\source\CMakeFiles\generate.stamp is up-to-date.
8>Generate info tables for SPIR-V vunified1 core instructions and operands.
8>Traceback (most recent call last):
8>  File "D:/projects/SPIRV-Tools-src/utils/generate_grammar_tables.py", line 749, in <module>
8>    main()
8>  File "D:/projects/SPIRV-Tools-src/utils/generate_grammar_tables.py", line 612, in main
8>    import argparse
8>ImportError: No module named argparse
```

**Comment by s-perron** [comment]
I've done this in #2424, where we make explicit that Python 2 is required, but we need to make sure it work in all of the environments that we care about.

**Comment by s-perron** [comment]
This might change when we fix #1856.
